main.js

```diff
@@ -1,8 +1,11 @@
 
 var $car = document.querySelector('.car');
 var carData = {
-  direction: 'east'
+  direction: 'east',
+  location: ['', '']
```

Comment:
location is much more readable when its an other object with x and y properties.